pkg/kds/context/testdata/full_sync/mesh-service/global.input.yaml

```diff
@@ -0,0 +1,26 @@
+---
+type: Mesh
+name: default
+---
+type: MeshService
+mesh: default
+name: redis-zone-2-ff29vc4685c6f594
```

Comment:
why did you put already "synced" resource to the `*.input.yaml` file? isn't the point of these tests to fill input files only with the resources that originate there and see in golden files they're synced correctly?

Reply:
I put it here so I can see that resource is synced to zone-1 and doesn't remove status field